File: src/com/uipkg/components/ListenerConfigurator.java
package com.uipkg.components;

import java.awt.event.AWTEventListener;
import java.util.ArrayList;

public class ListenerConfigurator {
  ArrayList<AWTEventListener> listeners = new ArrayList<>();

  public ListenerConfigurator(AWTEventListener... listeners) {
    for (AWTEventListener listener : listeners) {
      this.listeners.add(listener);
    }
  }

  public ArrayList<AWTEventListener> getListeners() {
    return listeners;
  }
}

File: src/com/uipkg/components/ListenerConfigurator.kt
package com.uipkg.components

import java.awt.event.AWTEventListener
import java.util.ArrayList

class ListenerConfigurator(vararg listeners: AWTEventListener) {
    var listeners = ArrayList<AWTEventListener>()

    init {
        for (listener in listeners) {
            this.listeners.add(listener)
        }
    }
}